{{cookiecutter.repo_name}}/docker-compose.yml

```diff
@@ -13,13 +13,11 @@ memcache:
   # Uncomment this line to use the official {{cookiecutter.repo_name}} base image
   image: {{cookiecutter.repo_name}}:v1
 
-  # Uncomment the next two lines to build from a local configuration repo
-  #build: ../configuration
-  #dockerfile: docker/build/{{cookiecutter.repo_name}}/Dockerfile
-
   container_name: {{cookiecutter.repo_name}}
   volumes:
     - .:/edx/app/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}
-  command: /edx/app/{{cookiecutter.repo_name}}/devstack.sh start
+  command: bash -c 'gunicorn --reload --workers=2 --name {{cookiecutter.repo_name}} -b :8360 -c /edx/app/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/docker_gunicorn_configuration.py --log-file - --max-requests=1000 {{cookiecutter.repo_name}}.wsgi:application'
```

Comment:
cookie cutter port should be used here i think? instead of 8360

Reply:
Though might need to be the string "1" + cookiecutter.port if you want to adhere to the current port structure we do for prod vs dev

i don't feel strongly about it, though if we do use a separate port for dev here, we'll probably also want to expose that dev port in the dockerfile as well. looks like there is not a build stage for dev in the dockerfile though, so i'm not sure if that's something you were looking to solve for

Reply:
After some clarification with the SRE team, it was decided that the port numbers should be 18xxx for both devstack and prod, so I updated to match that.